ui/locales/app.en.json

```
@@ -359,13 +340,20 @@
   "vox_filter_filtering_on": "Filtering on: ",
   "vox_filter_hydraulic_conductivity": "Hydraulic Conductivity",
   "vox_filter_lithology": "Modelling Unit",
+  "vox_filter_klasse": "Klasse",
   "vox_filter_max": "Max",
   "vox_filter_min": "Min",
   "vox_filter_or": "OR",
   "vox_filter_select_all": "Select all",
   "vox_filter_undefined_conductivity": "Undefined Conductivity",
   "vox_filter_undefined_lithology": "Undefined",
   "vox_filter_unselect_all": "Unselect all",
+  "vox_filter_klasse_1": "very large",
+  "vox_filter_klasse_2": "large",
+  "vox_filter_klasse_3": "moderately large ",
+  "vox_filter_klasse_4": "mämoderatessig",
+  "vox_filter_klasse_5": "low",
+  "vox_filter_klasse_6": "svery low",
```

Ich denke mal das ist ein Typo?
```suggestion
  "vox_filter_klasse_6": "very low",
```